consensus/polybft/fsm_test.go

```diff
@@ -285,7 +287,8 @@ func TestFSM_BuildProposal_WithoutUptimeTxGood(t *testing.T) {
 	checkpointBackendMock.On("BuildEventRoot", mock.Anything, mock.Anything).Return(types.ZeroHash, nil).Once()
 
 	fsm := &fsm{parent: parent, blockBuilder: mBlockBuilder, config: &PolyBFTConfig{}, backend: &blockchainMock{},
```

Comment:
A bit out of the scope but, thinking on having a constructor for the FSM...

Reply:
There are quite a few fields on the FSM to be initialized, however not all of them are required for all the tests. Therefore I don't clearly see benefit of introducing a constructor fn for it.